bootcamp/tutorials/quickstart/quickstart.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    # data: ["[{'id': 2, 'distance': 0.5859944820404053, 'entity': {'text': 'Born in Maida Vale, London, Turing was raised in southern England.', 'subject': 'history'}}, {'id': 1, 'distance': 0.5118255019187927, 'entity': {'text': 'Alan Turing was the first person to conduct substantial research in AI.', 'subject': 'history'}}]"] , extra_info: {'cost': 0}
</pre><p>No need to add this kind of comment in notebook. Just show the result in output cell. Please fix everywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1328/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
solved